lib/virtio/virtio.c

```diff
@@ -79,3 +79,44 @@ void virtio_describe(struct virtio_device *dev, const char *msg,
 	// TODO: Not used currently - keeping it for future use
 	virtio_feature_name(0, desc);
 }
+
+int virtio_create_virtqueues(struct virtio_device *vdev, unsigned int flags,
+			     unsigned int nvqs, const char *names[],
+			     vq_callback *callbacks[])
```

Comment:
should we be passing vring_info & vrings_num in, rather than holding onto them in the virtio_device?

Reply:
I was wondering if there was some value in having an API to create multiple queue's vs creating one queue at a time, and calling _create_virtqueue per queue that needs to be created.

Reply:
the previous create_virtqeueus() is one API to create multiple queues. this patch tries to move it to the virtio